sponsor-dapp-v2/src/components/Step6.js

```diff
@@ -25,6 +25,21 @@ function Step6(props) {
       .muln(100)
   );
 
+  // Add token to users metamask wallet.
+  window.web3.currentProvider.sendAsync({
+    method: "wallet_watchAsset",
+    params: {
+      type: "ERC20",
+      options: {
+        address: contractAddress,
+        symbol: symbol.substring(0, 6),
+        decimals: 18,
+        image: "https://umaproject.org/assets/images/UMA_square_red_logo.png"
+      }
+    },
+    id: Math.round(Math.random() * 100000)
```

Comment:
What's this id?

Reply:
Checked out metamask's website, and it looks like this is the suggested way to do this. Feel free to disregard.